Chapters/ChapterTwo.tex

```diff
@@ -46,14 +46,208 @@ \section{Robotics Frameworks}
+	\includegraphics[width=0.7\textwidth]
+	{Figures/Kumamoto_model.png}
+	\caption{\label{fig:Kumamoto_model} Kumamoto.}
+	\end{figure}
+
+\hspace{0.5cm} O hardware do robô foi bastante simplificado, visando facilitar o desenvolvimento pelos estudantes ainda não familiarizados com a robótica e eletrônica, além de buscar reduzir os custos de produção do robô. Como ferramenta educativa, o projeto conseguiu que seus estudantes produzissem o \textit{micromouse} em um semestre de atividades. Contudo, conceitos da robótica (ex: robótica móvel, fusão de sensores, visão, navegação) não foram trabalhados ou não foram citados no artigo gerado a partir do projeto.
+
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.7\textwidth]
+	{Figures/Kumamoto.png}
+	\caption{\label{fig:Kumamoto} Especificações do Kumamoto.}
+\end{figure}
+
+
+\textbf{Pontos Positivos:}
+\begin{itemize}
+	\item Projeto com fins educacionais;
+	\item Fácil desenvolvimento;
+\end{itemize}
+
+\textbf{Pontos Negativos:}
+\begin{itemize}
+	\item Não possui suporte nativo para ambiente ROS;
+	\item Pouca variedade de sensores;
+	\item Não possui guia para usuário;
+	\item Poucos recursos na interface com o usuário;
+	\item Não possui nenhuma ferramenta de simulação;
+\end{itemize}
+
+
+\subsection{WolfieMouse}
+\hspace{0.5cm} O WolfieMouse é um projeto de robótica que desenvolveu um \textit{micromouse} para competir na \textit{2018 Region 1 Robotics Competition}.
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.7\textwidth]
+	{Figures/WolfieMouse_model.jpg}
+	\caption{\label{fig:WolfieMouse_model} WolfieMouse.}
+\end{figure}
+
+\hspace{0.5cm} Além da plataforma robótica, que conta com hardwares programados em baixo nível, para melhor otimização de seus controles, a equipe também realizou um ambiente de simulação baseado em C++ emulado no terminal do computador. Toda documentação foi disponibilizada em um repositório git, que também possui tutoriais para o desenvolvimento do robô.
+
+\begin{figure}[H]
+	\centering
+	\includegraphics[width=0.7\textwidth]
+	{Figures/WolfieMouse.png}
+	\caption{\label{fig:WolfieMouse} Especificações do WolfieMouse.}
+\end{figure}
+
+\textbf{Pontos Positivos:}
+\begin{itemize}
+	\item Projeto bem documentado;
+	\item Possui tutoriais;
+	\item Possui ambiente de simulação;
+\end{itemize}
+
+\textbf{Pontos Negativos:}
+\begin{itemize}
+	\item Não possui suporte nativo para ambiente ROS;
+	\item Pouco foco em finalidades educativas com o produto;
+	\item Não possui guia para usuário;
+	\item Poucos recursos na interface com o usuário;
+\end{itemize}
+
+\subsection{Raspberry Pi Mouse V2}
+\hspace{0.5cm}A RT Corporation Micromouse é uma desenvolvedora japonesa de plataformas robóticas voltada para aplicações  voltadas de pesquisas à hobistas. Um de seus segmentos é voltado para \textit{micromouse}, fortemente representado pelo seu produto Raspberry Pi Mouse V2, citado em "Learning ROS robot programming with Raspberry Pi" (Nikkei BP, June 2018).
```

Comment:
```
fortemente representado pelo seu produto Raspberry Pi Mouse V2, citado em "Learning ROS robot programming with Raspberry Pi" (Nikkei BP, June 2018).
```
Acho que seria bom explicar superficialmente para que serve o livro.
Uma outra coisa, no final do parágrafo você está querendo citar uma fonte ou querendo falar quem foi o autor do livro? Se for a última opção, acho que ficaria melhor "do autor Nikkel BP". Mas se for citação, verificar regras conforme ABNT

Reply:
Não quis fazer nenhuma citação de conte ainda, pq teno que ver exatamente com ofaz. Deixei elas soltas para depois mexer nelas

Reply:
Dá uma olhada na branch do pré-projeto. Tem citação direta e indireta lá! Dá uma lida também em bibtex